t/lib/OpenQA/Test/Utils.pm

```diff
@@ -183,10 +183,10 @@ sub create_webapi {
 }
 
 sub create_websocket_server {
-    my ($port, $bogus, $nowait, $noworkercheck, $with_embedded_scheduler) = @_;
+    my ($port, $bogus, $nowait, $with_embedded_scheduler) = @_;
```

Comment:
Might be nice to use named arguments here since `0, 1, 1` reads a bit weird. No strong opinion, though.

Reply:
yes, but Martchus already made it nicer with one less parameter so it's good either way :)